lumigator/frontend/src/sdk/settingsService.ts

```diff
@@ -0,0 +1,42 @@
+import { lumigatorApiAxiosInstance } from '@/helpers/lumigatorAxiosInstance'
+import type { Secret, SecretUploadPayload } from '@/types/Secret'
+
+const PATH_SETTINGS_ROOT = 'settings'
+const PATH_SECRETS_ROOT = `${PATH_SETTINGS_ROOT}/secrets`
+const PATH_SECRET = (secretName: string) => `${PATH_SECRETS_ROOT}/${secretName}`
+
+/**
+ * Deletes a secret by name.
+ * @param {string} secretName - The name of the secret.
+ */
+export async function deleteSecret(secretName: string): Promise<void> {
+  return lumigatorApiAxiosInstance.delete(PATH_SECRET(secretName))
```

Comment:
no need for `async` here if we're returning the promise directly without `await`ing it